Export the show_task() function in order to get better process stacks.
It will also print the current process stack, which is useful.

This is a nice-to-have but not required for Lustre to work.

Index: linux-2.6.18-194.3.1-head/kernel/sched.c
===================================================================
--- linux-2.6.18-194.3.1-head.orig/kernel/sched.c
+++ linux-2.6.18-194.3.1-head/kernel/sched.c
@@ -5052,6 +5052,7 @@ void sched_show_task(struct task_struct 
 
 	show_stack(p, NULL);
 }
+EXPORT_SYMBOL(sched_show_task);
 
 void show_state(void)
 {
